templates/default/update-llnw.json.erb

```diff
@@ -1,51 +1,33 @@
 {
   "source_dir": "<%= @source_dir %>",
   "cdn_list": [
-    "llnw",
     "akamai",
     "edgecast",
     "cdnetworks"
-    ],
-    "ftp": {
-      "username" : "<%= @llnw_ftp_username %>"
-     ,"password" : "<%= @llnw_ftp_password %>"
-     ,"url"      : "<%= @llnw_ftp_url      %>"
-    },
-  "llnw": {
-    "ftp": {
-      "username" : "<%= @llnw_ftp_username %>"
-     ,"password" : "<%= @llnw_ftp_password %>"
-     ,"url"      : "<%= @llnw_ftp_url      %>"
-    },
-     "shortname"   : "<%= @llnw_shortname  %>"
-     ,"username"    : "<%= @llnw_username   %>"
-     ,"password"    : "<%= @llnw_password   %>"
-     ,"url"         : "<%= @llnw_url        %>"
-     ,"prepend_dir" : "<%= @llnw_prepend_dir %>"
-  },
+  ],
   "akamai": {
-   "ftp": {
-      "username" : "<%= @akamai_ftp_username %>"
-     ,"password" : "<%= @akamai_ftp_password %>"
-     ,"url"      : "<%= @akamai_ftp_url      %>"
+    "ftp": {
+      "username": "<%= @akamai_ftp_username %>",
+      "password": "<%= @akamai_ftp_password %>",
+      "url": "<%= @akamai_ftp_url      %>"
```

Comment:
only a single space around the var inside the erb block here would make sense if we are formatting the rest of the file too.